pkgs/by-name/t2/t2sz/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  zstd,
+}:
+
+stdenv.mkDerivation rec {
```

Lets use `finalAttrs` nixpkgs pattern instead of `rec` to avoid its drawbacks!
- #315337
```suggestion
stdenv.mkDerivation (finalAttrs: {
```